### PR TITLE
Update PyPI package documentation for v22.06.00

### PR DESCRIPTION
**Pull request by gigony**
- Update CHANGELOG.md and README.md for PyPI v22.06.00 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,37 @@
...
+- Merge branch-22.04 into branch-22.06 ([#258](https://github.com/rapidsai/cucim/pull/258)) [@jakirkham](https://github.com/jakirkham)
+
+## 📖 Documentation
+
+- update outdated links to example data ([#289](https://github.com/rapidsai/cucim/pull/289)) [@grlee77](https://github.com/grlee77)
+- Add missing API docs ([#275](https://github.com/rapidsai/cucim/pull/275)) [@grlee77](https://github.com/grlee77)
+
+## 🚀 New Features
+
+- add missing `cucim.skimage.segmentation.clear_border` function ([#267](https://github.com/rapidsai/cucim/pull/267)) [@grlee77](https://github.com/grlee77)
```
Comment (grlee77) [inline CHANGELOG.md:23]:
```suggestion
- add missing `cucim.skimage.segmentation.clear_border` function ([#267](https://github.com/rapidsai/cucim/pull/267)) [@grlee77](https://github.com/grlee77)
- add `cucim.core.operations.color.stain_extraction_pca` and `cucim.core.operations.color.normalize_colors_pca` for digital pathology H&E stain extraction and normalization ([#273](https://github.com/rapidsai/cucim/pull/273)) [@grlee77](https://github.com/grlee77), [@drbeh](https://github.com/drbeh)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,9 +2,37 @@
...
+
+## 🛠️ Improvements
+
+- Update to use DLPack v0.6 ([#295](https://github.com/rapidsai/cucim/pull/295)) [@gigony](https://github.com/gigony)
+- Remove plugin-related messages temporarily ([#291](https://github.com/rapidsai/cucim/pull/291)) [@gigony](https://github.com/gigony)
+- Simplify recipes ([#286](https://github.com/rapidsai/cucim/pull/286)) [@Ethyling](https://github.com/Ethyling)
+- Use cupy.fuse to improve efficiency hessian_matrix_eigvals ([#280](https://github.com/rapidsai/cucim/pull/280)) [@grlee77](https://github.com/grlee77)
+- Promote small integer types to single rather than double precision ([#278](https://github.com/rapidsai/cucim/pull/278)) [@grlee77](https://github.com/grlee77)
+- improve efficiency of histogram-based thresholding functions ([#276](https://github.com/rapidsai/cucim/pull/276)) [@grlee77](https://github.com/grlee77)
+- Remove unused dependencies in GPU tests job ([#268](https://github.com/rapidsai/cucim/pull/268)) [@Ethyling](https://github.com/Ethyling)
```
Comment (grlee77) [inline CHANGELOG.md:33]:
```suggestion
- Enable footprint decomposition for morphology ([#274](https://github.com/rapidsai/cucim/pull/274)) [@grlee77](https://github.com/grlee77)
- Remove unused dependencies in GPU tests job ([#268](https://github.com/rapidsai/cucim/pull/268)) [@Ethyling](https://github.com/Ethyling)
```

Comment (grlee77) [reply CHANGELOG.md:33]: added a missing PR here

File: CHANGELOG.md (hunk 1 above)
Comment (grlee77) [reply CHANGELOG.md:23]: stain extraction and normalization are also new

**Review by grlee77 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (gigony) [reply CHANGELOG.md:23]: Thanks! Updated.

File: CHANGELOG.md (hunk 2 above)
Comment (gigony) [reply CHANGELOG.md:33]: Thanks! Updated.

**Comment by grlee77** [comment]
rerun tests

**Comment by jakirkham** [comment]
@gigony could you please fix up the conflicts here?

**Comment by grlee77** [comment]
> @gigony could you please fix up the conflicts here?

It was only a couple of added lines, so I went ahead and resolved it from the web UI.

**Review by grlee77 — APPROVED** [review]
LGTM

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
@gpucibot merge

**Comment by jakirkham** [comment]
Thanks all! 🙏

**Pull request merged**